user-interface/src/lib/components/CaseNumber.tsx

```diff
@@ -18,7 +18,7 @@ export function CaseNumber(props: CaseNumberProps) {
     </span>
   );
   if (renderAs === 'link') {
-    const target = openLinkIn === 'new-window' ? '_blank' : '_self';
+    const target = openLinkIn === 'new-window' ? `CAMS-case-detail-${caseId}` : '_self';
```

Comment:
Why do we use a named target here?

Reply:
It opens a given case in the same window/tab each time. I think I'm going to revert it though because it felt awkward.